tutorials/guide-to-breaches.md

```diff
@@ -12,7 +12,9 @@ There are two kinds of breaches: **personal breaches** and **network breaches**.
 
 Ships on the Arvo network sometimes need to reset their continuity. A personal breach is when an individual ship announces to the network: "I forgot who I am, let's start over from scratch." That is, it clears its own event log and sends an announcement to the network, asking all ships that have communicated with it to reset its networking information in their state. This makes it as though the ship was just started for the first time again, since everyone has network has forgotten about it.
 
-Personal breaches often fix connectivity issues, but should only be used as a last resort. To perform a personal breach, follow the steps below.
+Personal breaches often fix connectivity issues, but should only be used as a last resort. Before performing a personal breach, reach out for help in the #ship-starting-support channel in our [Discord server](https://discord.gg/n9xhMdz) to see if there is another option, or look at alternative fixes in the [Ship Troubleshooting](../ship-troubleshooting) guide.
```

Comment:
Should this not maybe be:

> reach out for help in `~dopzod/urbit-help` or, failing that, the `#ship-starting-support` channel in our [Discord server](#)

? Or is that considered too convoluted?

Reply:
I am guessing that most people resorting to a personal breach can't connect to `~dopzod`, but I do see your point